lib/components/src/treeview/treeview.js

```diff
@@ -125,9 +125,7 @@ const calculateTreeProps = memoize(50)((input, filter, selectedId, extraExpanded
     {}
   );
 
-  const selectedAncestorIds = selectedId
-    ? getParents({ id: selectedId, dataset }).map(i => i.id)
-    : [];
+  const selectedAncestorIds = selectedId ? getParents(selectedId, dataset).map(i => i.id) : [];
```

Comment:
What's going on on this line? Is there a prettier issue somewhere between you and me? ;)